python/lsst/pipe/analysis/coaddAnalysis.py

```diff
@@ -251,22 +259,23 @@ def runDataRef(self, patchRefList, subdir="", cosmos=None):
             unforced = self.calibrateCatalogs(unforced, wcs=repoInfo.wcs)
             plotKwargs.update(dict(zpLabel=self.zpLabel))
             if haveForced:
-                # copy over some fields from unforced to forced catalog
-                forced = addColumnsToSchema(unforced, forced,
-                                            [col for col in list(self.config.columnsToCopy) +
-                                             list(self.config.analysis.flags) if
-                                             col not in forced.schema and col in unforced.schema and
-                                             not (repoInfo.hscRun and col == "slot_Centroid_flag")])
```

Comment:
I think this would be more legible and easier to understand if the list was defined prior to the function call.
 eg. colsTocopy = [col for col in ....], addColumnsToSchema(unforced, forced, colsToCopy).
Where is columnsToCopy defined? Was it not replaced by the new config option? I think that both of the config values are already lists and probably don't need casting to list.

Reply:
Done.  The list needs to be created dynamically to accommodate any columns that may not exist for a given repo (certain algs not run, for example) and we don't know a-priori what filters exist for the `merge_measurement_FILTER` flags).